src/js/undergrad.js

```diff
@@ -1,10 +1,10 @@
 const editModal = document.getElementById("editModal");
 const moduleEditBtn = document.getElementById("editModule");
-const modalDiscardBtn = document.getElementById("editModDiscardBtn");
+const modalDiscardBtn = document.getElementById("emDiscardBtn");
 
 const addModal = document.getElementById("addModal");
-const moduleAddBtn = document.getElementById("addModuleBtn");
-const modalCancelBtn = document.getElementById("addModCancelBtn");
+const createModBtn = document.getElementById("addModuleBtn");
+const modalCancelBtn = document.getElementById("amCancelBtn");
```

Comment:
I don't know if I am fan of these abbreviations. I understand the thought of it being quite long winded but personally I'd actually just prefer something like:
```javascript
const modalDiscardBtn = document.getElementById("editModalDiscardBtn")
const modalCancelBtn = document.getElementById("addModalCancelBtn")
```
I think the prefixes we have of edit and add are not so long that it makes these too bad in long form.